specification/appconfiguration/data-plane/Microsoft.AppConfiguration/preview/2023-05-01-preview/appconfiguration.json

```diff
@@ -0,0 +1,2079 @@
+          "GetSnapshot_IfNoneMatch": {
+            "$ref": "./examples/GetSnapshot_IfNoneMatch.json"
+          },
+          "GetSnapshot_Failed": {
+            "$ref": "./examples/GetSnapshot_Failed.json"
+          }
+        }
+      },
+      "put": {
+        "tags": [
+          "Snapshots"
+        ],
+        "summary": "Creates a key-value snapshot.",
+        "operationId": "CreateSnapshot",
+        "consumes": [
+          "application/vnd.microsoft.appconfig.snapshot+json",
+          "application/json"
+        ],
+        "produces": [
+          "application/vnd.microsoft.appconfig.snapshot+json",
+          "application/problem+json"
+        ],
+        "parameters": [
+          {
+            "name": "name",
+            "in": "path",
+            "description": "The name of the key-value snapshot to create.",
+            "required": true,
+            "type": "string",
+            "maxLength": 256
+          },
+          {
+            "name": "entity",
+            "in": "body",
+            "description": "The key-value snapshot to create.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/Snapshot"
+            }
+          },
+          {
+            "$ref": "#/parameters/SyncTokens"
+          },
+          {
+            "$ref": "#/parameters/ApiVersion"
+          }
+        ],
+        "responses": {
+          "201": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/Snapshot"
+            },
+            "headers": {
+              "Sync-Token": {
+                "description": "Enables real-time consistency between requests by providing the returned value in the next request made to the server.",
+                "type": "string"
+              },
+              "ETag": {
+                "description": "An identifier representing the returned state of the resource.",
+                "type": "string"
+              },
+              "Link": {
+                "description": "Includes links to related resources.",
+                "type": "string"
+              },
+              "Operation-Location": {
+                "description": "The URL to track the status of the long running operation.",
+                "type": "string"
+              }
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed",
+            "schema": {
+              "$ref": "#/definitions/Error"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "CreateSnapshot": {
+            "$ref": "./examples/CreateSnapshot.json"
+          }
+        },
+        "x-ms-long-running-operation": true
+      },
+      "patch": {
+        "tags": [
+          "Snapshots"
+        ],
+        "summary": "Updates the state of a key-value snapshot.",
+        "operationId": "UpdateSnapshot",
+        "consumes": [
+          "application/json"
```

Comment:
Patch should consume `application/merge-patch+json`.

Reply:
Updated, thanks!